CHANGELOG.md

```diff
@@ -1,5 +1,309 @@
+- Fabric: Removing accidental unnecessary BUCK dep ([18423fe](https://github.com/facebook/react-native/commit/18423fe) by [@shergin](https://github.com/shergin))
+- Prepend passed sourceExts to default ones and pass them to metro (#21855) ([ce86080](https://github.com/facebook/react-native/commit/ce86080) by [@elyalvarado](https://github.com/elyalvarado))
+- Wrap measureLayoutRelativeToContainingList in try-catch to mitigate crash ([5803772](https://github.com/facebook/react-native/commit/5803772))
+- RN: Missing Copyright Headers ([a689711](https://github.com/facebook/react-native/commit/a689711) by [@yungsters](https://github.com/yungsters))
+- Use fb_native_wrapper for all targets ([c147073](https://github.com/facebook/react-native/commit/c147073) by [@scottrice](https://github.com/scottrice))
+- gradle repo priority (#22041) ([2a349f8](https://github.com/facebook/react-native/commit/2a349f8) by [@dulmandakh](https://github.com/dulmandakh))
+- jest: upgrade to 24.0.0-alpha.4 ([66aba09](https://github.com/facebook/react-native/commit/66aba09) by [@rubennorte](https://github.com/rubennorte))
+- Flow strictifying AdsManagerAudienceImages.js ([136dfc8](https://github.com/facebook/react-native/commit/136dfc8))
+- BUCKFORMAT: apply on all of fbsource ([2b603fd](https://github.com/facebook/react-native/commit/2b603fd) by [@luciang](https://github.com/luciang))
+- Improved Types ([17fd1bc](https://github.com/facebook/react-native/commit/17fd1bc) by [@nmn](https://github.com/nmn))
+- Modified declarations from vars to const (#22070) ([efc9506](https://github.com/facebook/react-native/commit/efc9506) by [@fernando-sendMail](https://github.com/fernando-sendMail))
+- Deploy Flow v0.85 to xplat/js ([adc8a33](https://github.com/facebook/react-native/commit/adc8a33) by [@samwgoldman](https://github.com/samwgoldman))
+- CheckBox: Convert NativeMethodsMixin to forwardedRef, convert to class (#21585) ([28de61e](https://github.com/facebook/react-native/commit/28de61e) by [@empyrical](https://github.com/empyrical))
+- more lint auto fixing ([10d41d4](https://github.com/facebook/react-native/commit/10d41d4) by [@sahrens](https://github.com/sahrens))
+- lint autofixes ([2486d12](https://github.com/facebook/react-native/commit/2486d12) by [@sahrens](https://github.com/sahrens))
+- React sync for revisions 4773fdf...bf9fadf ([8b275a8](https://github.com/facebook/react-native/commit/8b275a8) by [@yungsters](https://github.com/yungsters))
+- Trivial cleanup in ReactRootView ([83c7303](https://github.com/facebook/react-native/commit/83c7303) by [@mdvacca](https://github.com/mdvacca))
+- Improving Modal `visible` prop check to handle undefined and null (#22072) ([cc13a73](https://github.com/facebook/react-native/commit/cc13a73) by [@MateusAndrade](https://github.com/MateusAndrade))
+- Flow strict ScrollViewMock (#22103) ([499c195](https://github.com/facebook/react-native/commit/499c195) by [@exced](https://github.com/exced))
```

Comment:
Internal only

Reply:
We could probably just roll all of these up into: "Improved Flow Typing for Core Components"